### ISSUE TITLE
OMP_THREAD_LIMIT does not limit number of threads used by tesseract

### ISSUE THREAD

**Issue by sebastianiorga** [jbarlow83/OCRmyPDF#716]
**Describe the bug**
What's the problem?
I'm calling `OMP_THREAD_LIMIT=1 ocrmypdf --deskew --rotate-pages --rotate-pages-threshold 5 -l lang input output` and don't want tesseract to be using multiple cores, but it's still using the full 8 on the machine.

I was assuming based on https://ocrmypdf.readthedocs.io/en/latest/advanced.html?highlight=omp_#envvar-OMP_THREAD_LIMIT that it would. 

**System (please complete the following information):**
 - OS: Ubuntu 20.04
 - Python version: 3.8.5
 - OCRmyPDF version: 9.6.0+dfsg

```
✗ tesseract --version
tesseract 4.1.1
 leptonica-1.79.0
  libgif 5.1.4 : libjpeg 8d (libjpeg-turbo 2.0.3) : libpng 1.6.37 : libtiff 4.1.0 : zlib 1.2.11 : libwebp 0.6.1 : libopenjp2 2.3.1
 Found AVX2
 Found AVX
 Found FMA
 Found SSE
 Found libarchive 3.4.0 zlib/1.2.11 liblzma/5.2.4 bz2lib/1.0.8 liblz4/1.9.2 libzstd/1.4.4
```

**Comment by jbarlow83** [comment]
There was a bug fixed in v9.7.0 where the message `Using Tesseract OpenMP thread limit %d` would show the wrong number. In practice (e.g. from htop) the correct number is actually used, the problem is just with the message.

Try the latest version. If you're still getting the issue trying Tesseract on an image directly (`OMP_THREAD_LIMIT=1 tesseract test.png stdout`) to see if OMP_THREAD_LIMIT takes effect.

**Comment by sebastianiorga** [comment]
I'm a donkey, I should used the `-j` or `--jobs` option. What I was seeing was actually `ocrmypdf` processing multiple pages of the PDF in parallel.

Thank you for the library, it's really cool! Sorry for wasting your time.
